src/ui/server/middleware/insurance/application-access/index.test.ts

```diff
@@ -84,6 +84,24 @@ describe('middleware/insurance/application-access', () => {
       });
     });
 
+    describe('when res.locals.application does not have an exporter ID', () => {
+      beforeEach(() => {
+        res.locals.application = {
+          ...mockApplication,
+          // @ts-ignore
```

Comment:
Note - this is required. Exporter ID is a required type in the Application type. So we cannot have an empty exporter object.

However in the dev environment, this has been the case, where an application has an empty exporter object. This might be from old applications before the exporter relationship was introduced.

These changes makes sure that this won't happen and we need to add a unit test for this.